### ISSUE TITLE
Can I use a single leaf train and expect two leaf test set?

### ISSUE THREAD

**Issue by afshinamiree** [dmlc/xgboost#1488]
I have the data  of my customers some name has been duplicated into several CustomerID e.g. name Young has three ID s of 300, 286, and 355
this may be three different customers with same name or there is a probability that Young has registered himself two or three times.
base of the unique names I can find   unique customers and I want to use this as train and find out if a customer for example with name Young in thee test set  is unique or not .
Thank you Very much

**Comment by wongjingping** [comment]
Hi @afshinamiree the issues list is for highlighting issues with xgboost, and isn't for getting help on formulating your problem. In any case if you're just interested in de-duplication an introduction to data wrangling might be helpful: https://ramnathv.github.io/pycon2014-r/explore/README.html
